### ISSUE TITLE
Introduce /docs folder to improve organization.

### ISSUE THREAD

**Issue by elg0nz** [aklepper20/AccessSphere#2]
# Notes:
Depends on #1 please finish that issue first before tackling this one 😺 

#### Problem: 
Our project's README.md structure will make it extremely long if we don't change approaches.
We propose introducing a `/docs` directory to simplify navigation.

#### Proposed Solution:
1. **Establish a `/docs` Directory:** Introduce a `/docs` directory in our README.md for detailed documentation, enhancing user-friendliness and navigation.
2. **Kickstart `./docs/architecture.md`:** Begin with an empty `./docs/architecture.md`, setting the stage for Systems Design exploration in our upcoming Mentorship session using the Radio framework ([Great Frontend - System Design Framework](https://www.greatfrontend.com/system-design/framework)).
3. **Relocate Login Flow Documentation:** Shift login flow details from README.md to `./docs/login-flow.md` for a more organized and accessible presentation.

#### Goals:
- Implement a structured `/docs` directory for organized, accessible documentation.
- Prepare for an engaging Systems Design mentorship session.
- Streamline README.md by moving detailed login flow info to `/docs`.

#### Non-Goals:
- Immediate content for `./docs/architecture.md` and `./docs/login-flow.md`.
- Overhauling unrelated README.md sections.

#### Acceptance Criteria:
- A user-friendly `/docs` directory integrated into README.md.
- `./docs/architecture.md` prepared for the Systems Design session.
- Login flow details are neatly organized in `./docs/login-flow.md`.
- Approval is required from a project maintainer for all changes.
